index.html

```diff
@@ -119,264 +119,156 @@
     </p>
     <p>
-      This flexibility permits an easy integration of new device
-      types and protocols, however it risks interoperability,
-      since there are <em>no guarantees</em> that two devices
-      which are formally spec-compliant, will be able to
-      communicate.
+      Future versions of this specification, or extension specifications, may 
+      define additional Profiles.
     </p>
-    <p>To increase adoption of the WoT specifications,
-      interoperability between on premise devices, edge devices
-      and the cloud is essential. Even if every manufacturer is
-      implementing the current Thing Description specification in
-      full flexibility, there is no interoperability guarantee;
-      many choices are still left to the implementations and there
-      are very few normative requirements that a device has to
-      fulfill.</p>
-
-    <section>
-      <h3>Deployment Scenarios</h3>
-
-      <p>A Thing Description can be used in two fundamentally
-        different deployment scenarios:</p>
-      <ul>
-        <li>a "brown-field" scenario, where it is created
-          to describe the interactions with existing systems.</li>
+  </section>
 
-        <li>a "green-field" scenario, where a device model
-          and a thing description are developed together.</li>
-      </ul>
+  <section id='sotd'></section>
 
+  <section id="introduction">
+    <h1>Introduction</h1>
+    <!-- Motivation -->
+    <section id="motivation">
+      <h2>Motivation</h2>
       <p>
-        For green field deployments, where the implementations
-        are being carried out and corresponding thing
-        descriptions are being created, it is easier to achieve
-        full interoperability by using a small, extensible <a href="#http-basic-profile">HTTP Basic
-          Profile</a>.
+        The <a href="https://www.w3.org/WoT/">Web of Things</a> (WoT) seeks to 
+        counter the fragmentation of the
+        <a href="https://en.wikipedia.org/wiki/Internet_of_things">Internet of 
+        Things</a> (IoT) by using and extending existing, standardized 
+        web technologies.
       </p>
-
-      <p>In the brown field area, due to the nature of
-        existing deployments and protocols, a broad spectrum of
-        variations and potentially high complexity of thing
-        descriptions inhibits interoperability and will most
-        likely lead to additional profiles of the <a>WoT Thing Description</a>
-        and domain-specific thing consumer implementations.</p>
-
       <p>
-        The WoT HTTP Basic Profile can be used by green field
-        deployments and gives guidance to new implementers of
-        the WoT specifications.
+        The W3C WoT Thing Description [[wot-thing-description]] specification
+        defines the first building block of the Web of Things, by defining an 
+        information model and JSON-based representation format for describing the
```

Comment:
```suggestion
        information model and JSON-based representation format. It can be used to describe the
```

Reply:
What's the reason for this change? I prefer the existing wording since it explains what the data model and representation format are for.